src/did/src/http.rs

```diff
@@ -0,0 +1,148 @@
+use std::borrow::Cow;
+use std::collections::HashMap;
+
+use candid::CandidType;
+use jsonrpc_core::{Error, Failure, Id, Version};
+use serde::de::DeserializeOwned;
+use serde::Deserialize;
+use serde_bytes::ByteBuf;
+use serde_json::Value;
+
+// A HTTP response.
+#[derive(Clone, Debug, CandidType, Deserialize)]
+pub struct HttpResponse {
```

Comment:
There is a duplicated definition of both the HttpRequest and HttpResponse in the `ethereum-json-rpc-client` crate, remove them and use the two here defined